tooling/cli/templates/plugin/with-api/examples/tauri-app/src-tauri/Cargo.toml

```diff
@@ -11,13 +11,13 @@ rust-version = "1.60"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [build-dependencies]
-tauri-build = {{ tauri_build_dep }}
+tauri-build = "1"
```

Comment:
instead of changing versions, and removing the template strings altogether, we can just rename the file to `Cargo.crate-manifest` and then modify the generator to change the name back, this is already done for https://github.com/tauri-apps/tauri/blob/3752eb1e6be2a8c31ea5519b5f1886d5721e4435/tooling/cli/templates/app/src-tauri/Cargo.crate-manifest